### PR TITLE
Blog post on owning identity and backups

### PR DESCRIPTION
**Pull request by ntolia**
## Description

Blog post on owning identity and backups

### CODE REVIEW

File: website/blog/2022-12-2-your-own-backups.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+So, ensure you and your data is always protected, set up regular backups! 
+
+# How can Corso help? 
+
+Backups can be complicated and, till recently, great backup solutions for Microsoft 365 might have been “enterprise” offerings that would cost many, many, MANY coffees a month. However, all of that is now behind us!  
+
+We recently introduced Corso, a free, secure, and 100% open-source tool to let you create a backup of all your data in your Microsoft subscription. Backups go to a cloud storage provider of your choice, giving you fine-grained control of your data and independence from your primary cloud provider. Data is also compressed and deduplicated to reduce cost (pennies a month with systems such as AWS Glacier Instant Access) and encrypted with user-provided keys so that no one can see your data and that it is safe against hackers and ransomware. 
+
+![corso Screenshot](../static/img/blog/corso_backup.png)
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:41]: Can you create a screenshot with larger fonts, please?

File: website/blog/2022-12-2-your-own-backups.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Corso is available now, and we’d love to hear your thoughts! Download it, give it a try, and join us on Discord to let us know what you think. 
+
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:44]:
```suggestion
```

File: website/blog/2022-12-2-your-own-backups.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+We recently introduced Corso, a free, secure, and 100% open-source tool to let you create a backup of all your data in
+your Microsoft subscription. Backups go to a cloud storage provider of your choice, giving you fine-grained control of
+your data and independence from your primary cloud provider. Data is also compressed and deduplicated to reduce cost
+(pennies a month with systems such as AWS Glacier Instant Access) and encrypted with user-provided keys so that no one
+can see your data and that it is safe against hackers and ransomware.
+
+![corso Screenshot](../static/img/blog/corso_backup.png)
+
+Corso is available now, and we’d love to hear your thoughts! Download it, give it a try, and join us on Discord to let
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:71]: Please link to Discord and the docs download page.

File: website/blog/2022-12-2-your-own-backups.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+title: "Owning Your Identity and Data For The Price of One ☕️/Month "
+authors: nica
+tags: [corso, microsoft 365]
+---
+
+
+I recently saw a tweet that really spoke to me:
+
+![Gian Paolo tweet stating The only long-term solution to maintain your identity is: 1. your own domain name 2. your own website/blog 3. several backups](../static/img/blog/gian_tweet_backups.png)
+
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:12]: Like the first blog post, please add a 
```
<!-- truncate -->
```

Here or wherever you want the preview to end?

**Review by ntolia — COMMENTED** [review]
Couple of minor changes. You can merge after this. Notion has instructions on how to publish the blog and you should see if that works for you.

File: website/blog/2022-12-2-your-own-backups.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+Thankfully, the growth of easy-to-use SaaS services now allow you to, for the price of a coffee a month,
+control and maintain not just your identity without risk but also all your personal and,
+for all you entrepreneurs, business data too. There’s a happy state, for individuals and teams, where you feel fully in
+control of your own identity. This makes you more resilient to systemic failures and changes in one platform’s policy.
+How do we achieve this ownership nirvana? The pieces are simple:
+
+* Buy your own domain: Right now, `.me` is available for $3/year and `.online` is available for $0.99/year
+
+* Pick someone to host your email with the above domain: We highly recommend [Microsoft 365 Business for $6 /month](https://www.microsoft.com/en-us/microsoft-365/business)
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:29]:
```suggestion
* Pick someone to host your email with the above domain: We highly recommend [Microsoft 365 Business for $6 /month](https://www.microsoft.com/en-us/microsoft-365/business).
```

File: website/blog/2022-12-2-your-own-backups.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+ (due to SSO via your email account), AdWords account suspensions lead to a small business unable to advertise, or a
+ flagged application account leads to published applications being automatically withdrawn from app stores.
+
+Thankfully, the growth of easy-to-use SaaS services now allow you to, for the price of a coffee a month,
+control and maintain not just your identity without risk but also all your personal and,
+for all you entrepreneurs, business data too. There’s a happy state, for individuals and teams, where you feel fully in
+control of your own identity. This makes you more resilient to systemic failures and changes in one platform’s policy.
+How do we achieve this ownership nirvana? The pieces are simple:
+
+* Buy your own domain: Right now, `.me` is available for $3/year and `.online` is available for $0.99/year
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:27]:
```suggestion
* Buy your own domain: Right now, `.me` is available for $3/year and `.online` is available for $0.99/year.
```

File: website/blog/2022-12-2-your-own-backups.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+* Pick someone to host your email with the above domain: We highly recommend [Microsoft 365 Business for $6 /month](https://www.microsoft.com/en-us/microsoft-365/business)
+
+* Microsoft Office applications that you know and love, 1TB of file storage, unlimited video meetings,
+and more: FREE with your Microsoft 365 subscription
```
Comment (ntolia) [inline website/blog/2022-12-2-your-own-backups.md:32]:
```suggestion
and more: FREE with your Microsoft 365 subscription.
```

**Review by serverless-mom — APPROVED** [review]

**Pull request merged**